packages/components/src/Badge/Badge.types.ts

```diff
@@ -0,0 +1,10 @@
+import { BaseComponentAttributes } from '@ids-types/general';
+
+export const BADGE_SIZE_VALUES = ['medium', 'small'] as const;
```

Comment:
If you want (and I know you want! :P ) you can change it to enum, maybe it would even be better as then we would be able to import these in other components and use BADGE_SIZE_VALUES.MEDIUM instead of declaring "medium" each time